sdk/textanalytics/Azure.AI.TextAnalytics/tests/DetectLanguageTests.cs

```diff
@@ -266,11 +266,27 @@ public void DetectLanguageBatchDisableServiceLogsThrows()
             Assert.AreEqual("TextAnalyticsRequestOptions.DisableServiceLogs is not available in API version v3.0. Use service API version v3.1 or newer.", ex.Message);
         }
 
+        [RecordedTest]
+        [ServiceVersion(Min = TextAnalyticsClientOptions.ServiceVersion.V2022_10_01_Preview)]
+        public async Task DetectLanguageBatchWithScriptTest()
+        {
+            string document = "Tumhara naam kya hai?";
+
+            TextAnalyticsClient client = GetClient();
+            TextAnalyticsRequestOptions options = new TextAnalyticsRequestOptions() { ModelVersion = "2022-04-10-preview" };
+
+            DetectLanguageResultCollection results = await client.DetectLanguageBatchAsync(new List<string>() { document }, options: options);
+
+            Assert.AreEqual(ScriptKind.Latin, results[0].PrimaryLanguage.Script);
```

Comment:
Perhaps dubious value since we're testing the service, but given other issues we've seen perhaps pass in another non-Latin script and verify we get back "other".